### PR TITLE
Added Pagination to the website #139

### PR DESCRIPTION
**Pull request by kritikaparmar-programmer**
Fixes: #139 

Description:
Added pagination to the website

# Screenshots
page

page1

### CODE REVIEW

**Comment by kritikaparmar-programmer** [comment]
Hey @harsh-9in and @Kajol-Kumari , I've added pagination, Please review the changes ..

File: mysite/blog/templates/blog/home.html [hunk 1]
```diff
@@ -54,4 +61,30 @@ <h2><a class="article-title" href="{% url 'post-detail' post.slug %}">{{ post.ti
...
+
+    {% if posts.has_previous %}
+    <a class="btn btn-outline-info mb-4" href="?page=1">First</a>
+    <a class="btn btn-outline-info mb-4" href="?page={{ posts.previous_page_number }}">Previous</a>
+    {% endif %}
+
+    {% for num in posts.paginator.page_range %}
+    {% if posts.number == num %}
+    <a class="btn btn-info mb-4" href="?page={{ num }}">{{ num }}</a>
+    {% elif num > posts.number|add:'-3' and num < posts.number|add:'3' %}
```
Comment (Kajol-Kumari) [inline mysite/blog/templates/blog/home.html:77]: Hey @kritikaparmar-programmer let's keep 2 blogs per page.

**Review by Kajol-Kumari — APPROVED** [review]
Minir change. Rest looks preety good to me :)

File: mysite/blog/templates/blog/home.html (hunk 1 above)
Comment (kritikaparmar-programmer) [reply mysite/blog/templates/blog/home.html:77]: As you say ...😁

Comment (kritikaparmar-programmer) [reply mysite/blog/templates/blog/home.html:77]: Done the changes..

**Pull request merged**